tests/circuit_drawer/test_circuit_drawer.py

```diff
@@ -847,6 +847,28 @@ def circuit():
 
         assert res == "\n".join(expected)
 
+    def test_include_empty_wires(self, tape_only):
+        """Test that empty wires are correctly included"""
+
+        dev = qml.device('default.qubit', wires=[-1, "a", "q2", 0])
+
+        @qml.qnode(dev)
+        def circuit():
+            qml.Hadamard(wires=-1)
+            qml.CNOT(wires=[-1, "q2"])
+            return qml.expval(qml.PauliX(wires="q2"))
+
+        circuit()
+        res = circuit.draw(show_all_wires=True)
+        expected = [
+            " -1: ──H──╭C──┤     ",
+            "  a: ─────│───┤     ",
+            " q2: ─────╰X──┤ ⟨X⟩ ",
+            "  0: ─────────┤     \n"
+        ]
+
+        assert res == "\n".join(expected)
```

Comment:
Would the zero wire be drawn at position 2 if it was at position 2 on the device? Or do they get sorted to the end? Maybe good to change the test data to this slightly more intricate case?

Reply:
Yep, the empty wires will match the device position (note wire `a` is also empty, and in position 1)

Reply:
Oh of course!